### DISCUSSION TITLE
roadmap/cpp/how_to_learn_cpp

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#38] [Announcements]
# roadmap/cpp/how_to_learn_cpp

如何系统学习C++语言，C++该怎么去学习，冲击大厂，C++ 应用场景，在大公司都有哪些岗位？

https://csguide.cn/roadmap/cpp/how_to_learn_cpp.html

**Comment by pnyyb** [comment]
https://www.codewars.com/kata/search/cpp博主你推荐的这个入门网站是在哪里学语法啊，怎么感觉是算法题

**Reply by pnyyb** [reply]
裂开了，推荐的都是英文，我好难学

**Comment by TiCiL** [comment]
👍
